tidb-cloud/setup-azure-self-hosted-kafka-private-link-service.md

```diff
@@ -0,0 +1,539 @@
+---
+title: Set Up Self-Hosted Kafka Private Link Service in Azure
+summary: This document explains how to set up Private Link service for self-hosted Kafka in Azure and how to make it work with TiDB Cloud.
+---
+
+# Set Up Self-Hosted Kafka Private Link Service in Azure
+
+This document describes how to set up Private Link service for self-hosted Kafka in Azure, and how to make it work with TiDB Cloud.
+
+The mechanism works as follows:
+
+1. The TiDB Cloud Vnet connects to the Kafka Vnet through private endpoints.
+2. Kafka clients need to communicate directly to all Kafka brokers.
+3. Each Kafka broker is mapped to a unique port of endpoints within the TiDB Cloud Vnet.
+4. Leverage the Kafka bootstrap mechanism and Azure resources to achieve the mapping.
+
+The following diagram shows the mechanism. 
+
+![Connect to Azure Self-Hosted Kafka Private Link Service](/media/tidb-cloud/changefeed/connect-to-azure-self-hosted-kafka-privatelink-service.jpeg)
```

Comment:
Please use an image in the .png format instead.

Reply:
<img width="629" alt="Screen Shot 2025-05-08 at 09 35 07" src="https://github.com/user-attachments/assets/8840319c-9322-4c2c-bfcd-e1e802bc281d" />
It seems we are using both png and jpeg now.